cocaine12/worker_handlers.go

```diff
@@ -0,0 +1,54 @@
+package cocaine12
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+)
+
+// EventHandler represents a type of handler
+type EventHandler func(context.Context, Request, Response)
+
+// RequestHandler represents a type of handler
+type RequestHandler func(context.Context, string, Request, Response)
+
+// TerminationHandler invokes when termination message is received
+type TerminationHandler func(context.Context)
+
+type EventHandlers struct {
```

Comment:
I think that "Handler" is enough for library user to make any needed implementation. I think there is no need for cocaine-lib to provide that kind of functionality. I would event delete this, but it will make harder to migrate to new interface.